pkg/newlog/cmd/newlogd.go

```diff
@@ -843,21 +845,29 @@ func getAppStatsMap(appuuid string) statsLogFile {
 		applogfile := startTmpfile(collectDir, applogname, true)
 
 		var notUpload bool
-		for _, appD := range domainUUID {
+		var appD appDomain
+		found := false
+		clb := func(key string, val interface{}) bool {
+			appD = val.(appDomain)
```

Comment:
Do we need this complexity? I think the original code didn't need a loop and could have just looked for domainUUID[appuuid], but please check with @naiming-zededa

Reply:
Make sense, fixed. Thank you for finding it out.

Reply:
@naiming-zededa can you please review the current implementation?